taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/controller/AndroidDriverController.java

```diff
@@ -142,6 +144,21 @@ public String getPageSource() {
         return androidDriver.getPageSource();
     }
 
+    @Override
+    public void inspectMemoryUsage(String targetApp, String description, boolean isReset) {
+        // TODO: Need to add memory stack profiling inspector here
+        PerformanceInspectionService.getInstance()
+                .inspect(PerformanceInspection.createAndroidMemoryInfoInspection(
+                        targetApp, this.udid, description, isReset));
```

Comment:
JFYI, the device identifier will be updated during the test run. So, we may not need to store the udid for T2C controllers. For more details, pls refer to the `PerformanceTestManagementService: : getDevicePerformanceInspection()`

Reply:
Will refactor this in the future with perf infra